### ISSUE TITLE
Jonathan L. - Build UI

### ISSUE THREAD

**Issue by KrakHub** [KrakHub/email-scraper#1]
You seem smart so you should have this easy.

Have a UI where you can input multiple domains, and a button to submit it to the main function (that Patrick is working on)

**Comment by KrakHub** [comment]
Looks good, you'll have more work to do once Blystone's UI is done, so I'll a new issue for you once Blystone is done. In the meantime, see if you can help him with the UI. We're making good progress, let's keep it up!
